### PR TITLE
COR-651: set correct default value for non-pointer time values

### PR DESCRIPTION
**Pull request by internetti**
while encoding, non-pointer Time values where not checked for having been initialized

### CODE REVIEW

File: pkg/views/forms/codec_time.go [hunk 1]
```diff
@@ -25,6 +25,9 @@ func (c *TimeCodec) Encode(value interface{}) (string, error) {
 		}
 		return v.Format(c.format), nil
 	case time.Time:
+		if v.IsZero() {
```
[the remark this answers is not in the document]
Comment (internetti) [reply pkg/views/forms/codec_time.go:28]: this exact line is showing an error, "time.Time" is not an expression

[the remark this answers is not in the document]
Comment (ludydoo) [reply pkg/views/forms/codec_time.go:28]: Maybe missing the `{}` in `v == time.Time{} || v.IsZero()`

[the remark this answers is not in the document]
Comment (internetti) [reply pkg/views/forms/codec_time.go:28]: no, I copied your exact line. I'll try again

[the remark this answers is not in the document]
Comment (internetti) [reply pkg/views/forms/codec_time.go]: it was missing some brackets

**Comment by neb42** [comment]
lgtm

**Pull request merged**